src/web-auth/captcha.js

```diff
@@ -161,6 +162,7 @@ function injectCaptchaScript(element, opts, callback, setValue) {
   );
   if (opts.provider === ARKOSE_PROVIDER) {
     var retryCount = 0;
+    window.requiresTrigger = true;
```

Comment:
Yeah we don't have other state information and object to store it. We could run through the captcha provider global objects but that would make unnecessarily ugly code. Although I think it might be more valuable in the future to save the provider name rather than the requires trigger boolean.